src/components/Icon/svgIcons.tsx

```diff
@@ -0,0 +1,17 @@
+import * as React from 'react'
+
+export interface ISvgIconSpec {
+  element: any
+  viewBox?: string
+}
+
+const svgIcons: { [name: string]: ISvgIconSpec } = {
+  umbrella: {
+    viewBox: '0 0 34 34',
```

Comment:
Should we have a default value for the viewBox? In Teams, not all, but most of the icons have the same viewBox which is '0 0 32 32'